install.sh

```diff
@@ -197,8 +199,9 @@ fi
 cd "${CREW_CONFIG_PATH}"
 echo_info "\nCreating device.json."
 jq --arg key0 'architecture' --arg value0 "${ARCH}" \
-  --arg key1 'installed_packages' \
-  '. | .[$key0]=$value0 | .[$key1]=[]' <<<'{}' > device.json
+  --arg key1 'last_update_check' --arg value1 "`date +%s`" \
+  --arg key2 'installed_packages' \
```

Comment:
Rubocop doesn't like backticks.

Reply:
Shellcheck!

Reply:
Fixed.

Reply:
Yes, shellcheck instead of rubocop since this is a shell script. :)